cmd/epp/runner/runner.go

```diff
@@ -209,6 +215,29 @@ func (r *Runner) Run() error {
 		return err
 	}
 
+	var theConfig *v1alpha1.EndpointPickerConfig
+	var instantiatedPlugins map[string]plugins.Plugin
+
+	if len(*configText) != 0 || len(*configFile) != 0 {
+		theConfig, err = config.LoadConfig([]byte(*configText), *configFile)
+		if err != nil {
+			setupLog.Error(err, "Failed to load the configuration")
+			return err
+		}
+
+		instantiatedPlugins, err = config.LoadPluginReferences(theConfig.Plugins, eppHandle{})
```

Comment:
```suggestion
                epp := eppHandle{}
		instantiatedPlugins, err = config.LoadPluginReferences(theConfig.Plugins, epp)
```
Make configuring the handle more explicit than as a stack based variable?